src/services/serviceResources/locationResource.json

```diff
@@ -0,0 +1,1031 @@
+{
```

Comment:
the only reason the location information was previously being dynamically retrieved from craigslist was to account for potential changes in location name (on craigslist's side) right? if so, i think having a static json file is good for the scope of this web dev project. i'm super glad the listings are still showing up.

Reply:
Yep